core/src/main/resources/hudson/views/JobColumn/column.jelly

```diff
@@ -26,6 +26,6 @@ THE SOFTWARE.
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
     <td style="${indenter.getCss(job)}">
         <!-- TODO consider using ${rootURL}/${job.url} instead (also in other column.jelly) -->
-        <a href="${jobBaseUrl}${job.shortUrl}" class='model-link inside'> <l:breakable value="${h.getRelativeDisplayNameFrom(job, itemGroup)}"/></a>
+        <a href="${jobBaseUrl}${job.shortUrl}" class='jenkins-table__link model-link inside'> <l:breakable value="${h.getRelativeDisplayNameFrom(job, itemGroup)}"/></a>
```

Comment:
Can't we reuse the existing classes? Otherwise we need to change every plugin to get it adapted to the new style.

Reply:
There aren't existing classes for a link inside of a table afaik, and styling `.jenkins-table a` could have unintended consequences where a developer wants a link that doesn't look like `.jenkins-table__link`.